### PR TITLE
refactor(send): Denomination label now matches selected currency unit

### PR DESCRIPTION
**Pull request by flaviomoceri**
### Description

Denomination label now matches selected currency unit

### Linked Issues/Tasks

https://app.asana.com/0/0/1205905791108240/f

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Refactoring (improving code without creating new functionality)

### Tests

- [ ] Detox test
- [ ] Unit test
- [X] No test

### Screenshot / Video

https://github.com/synonymdev/bitkit/assets/63161412/a015c6ac-ed58-4226-847e-0f735893953e

### CODE REVIEW

File: src/screens/Wallets/NumberPadButtons.tsx [hunk 1]
```diff
@@ -68,9 +68,9 @@ const NumberPadButtons = ({
 						onPress={onChangeUnit}>
 						<SwitchIcon color={color} width={16.44} height={13.22} />
 						<Caption13Up color={color} style={styles.middleButtonText}>
-							{nextUnit === 'BTC' && 'BTC'}
-							{nextUnit === 'satoshi' && 'sats'}
-							{nextUnit === 'fiat' && fiatTicker}
+							{nextUnit === 'BTC' && fiatTicker}
+							{nextUnit === 'satoshi' && 'BTC'}
+							{nextUnit === 'fiat' && 'sats'}
```
Comment (pwltr) [inline src/screens/Wallets/NumberPadButtons.tsx]: In this component we can use `unit` directly and get rid of `nextUnit` for a small cleanup.

```suggestion
							{unit === EUnit.BTC && 'BTC'}
							{unit === EUnit.satoshi && 'sats'}
							{unit === EUnit.fiat && fiatTicker}
```

**Review by pwltr — CHANGES_REQUESTED** [review]

File: src/screens/Wallets/NumberPadButtons.tsx (hunk 1 above)
Comment (flaviomoceri) [reply src/screens/Wallets/NumberPadButtons.tsx]: I made the changes, but perhaps I didn't understand your request. If necessary, provide me with more details please.

File: src/screens/Wallets/NumberPadButtons.tsx [hunk 2]
```diff
@@ -68,9 +57,9 @@ const NumberPadButtons = ({
 						onPress={onChangeUnit}>
 						<SwitchIcon color={color} width={16.44} height={13.22} />
 						<Caption13Up color={color} style={styles.middleButtonText}>
-							{nextUnit === 'BTC' && 'BTC'}
-							{nextUnit === 'satoshi' && 'sats'}
-							{nextUnit === 'fiat' && fiatTicker}
+							{unit === EUnit.BTC && fiatTicker}
+							{unit === EUnit.satoshi && 'BTC'}
+							{unit === EUnit.fiat && 'sats'}
```
Comment (pwltr) [inline src/screens/Wallets/NumberPadButtons.tsx]:
```suggestion
							{unit === EUnit.BTC && 'BTC'}
							{unit === EUnit.satoshi && 'sats'}
							{unit === EUnit.fiat && fiatTicker}
```

**Review by pwltr — CHANGES_REQUESTED** [review]

File: src/screens/Wallets/NumberPadButtons.tsx (hunk 2 above)
Comment (flaviomoceri) [reply src/screens/Wallets/NumberPadButtons.tsx]: But if I do this, the denomination label does not match to the selected monetary unit as requested in the changes made by Aldert or I am missing something?

In the other files I do the same thing:
```
{nextunit === EUnit.BTC && fiatTicker}
{nextunit === EUnit.satoshi && 'BTC'}
{nextunit === EUnit.fiat && 'sats'}
```

Comment (pwltr) [reply src/screens/Wallets/NumberPadButtons.tsx]: You didn't quite apply the suggestions correctly, please double-check. Maybe the confusion comes from `nextUnit` vs `unit`, those two are not the same.

Comment (flaviomoceri) [reply src/screens/Wallets/NumberPadButtons.tsx]: Yes, I probably don't understand something, however I made the requested changes, sorry about that

**Review by pwltr — APPROVED** [review]

**Pull request merged**